src/ui/Templates/TemplateToHtml.ts

```diff
@@ -0,0 +1,71 @@
+import { IQueryResults } from '../../rest/QueryResults';
+import { IQueryResult } from '../../rest/QueryResult';
+import { Assert, QueryUtils, Component, Initialization, QueryStateModel } from '../../Core';
+import { IInitResult } from '../Base/Initialization';
+import { Template } from './Template';
+import { SearchInterface } from '../../ui/SearchInterface/SearchInterface';
+import { ResultList } from '../ResultList/ResultList';
+import { ValidLayout } from '../ResultLayoutSelector/ValidLayout';
+
+export interface ITemplateToHtml {
+  resultTemplate: Template;
+  queryStateModel: QueryStateModel;
+  searchInterface: SearchInterface;
+}
+
+export class TemplateToHtml {
+  constructor(public args: ITemplateToHtml) {}
+
+  /**
+   * Builds and returns an array of HTMLElement with the given result set.
```

Comment:
Just FYI, the JSDoc here will never appear in any generated documentation, because the `TemplateToHtml` class itself is undocumented. 

This is okay though, what's important is the documentation on the top level `ResultList` or `QueryPreview` component.

So, I would remove it I guess.

Reply:
ok great will do